### PR TITLE
feat: add i18next

### PR DESCRIPTION
**Pull request by Yulatun**

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { i18n } from './i18n';
+
+export { i18n };
```
Comment (itsnotpolza) [inline src/i18n/index.js:3]:
```suggestion
export { i18n } from './i18n';
```

File: src/main.jsx [hunk 2]
```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
 import './index.css';
-import { ChakraProvider } from '@chakra-ui/react'
+import { ChakraProvider } from '@chakra-ui/react';
+import { i18n } from './i18n'; //eslint-disable-line
```
Comment (itsnotpolza) [inline src/main.jsx:6]: you can just
```suggestion
import './i18n';
```

**Comment by itsnotpolza** [comment]
Resolve conflicts with master branch

**Review by HarryPotter-4000 — DISMISSED** [review]

**Review by kalashnikau — APPROVED** [review]

**Review by lazzzaroni — APPROVED** [review]
I hope it works 🤞

**Pull request merged**